default_config/dev/config.yaml

```
@@ -1,8 +1,19 @@
 API_HOST_URL: 'http://localhost:8000'
+SSO_URL: 'http://localhost:8000'
+SSO_URL_SUCCESSFUL_LOGIN:  'http://localhost:8000'
 ENVIRONMENT_DISPLAY_NAME: 'Localhost'
 FEEDBACK_EMAIL_ADDRESS: 'support@example.com'
 OAUTH_CLIENT_ID: ''
 
+SSO_BUTTON_TITLE: {
+ar: "الدخول عبر SSO",
+en: "Sign in with SSO"
+}
+
+
 UI_COMPONENTS:
     COURSE_UNIT_PROGRESS_ENABLED: false
-    COURSE_NESTED_LIST_ENABLED: false
+    COURSE_NESTED_LIST_ENABLED: true
```

```suggestion
    COURSE_NESTED_LIST_ENABLED: false
```

That's what you mean @volodymyr-chekyrta, right?

---

Yes 👍